etc/tendrl_definitions_node_agent.yaml

```diff
@@ -20,26 +20,23 @@ object_details:
     enabled: True
```

Comment:
Need "package" object (to fetch list of rpms installed, yum repos etc) and "service" object (to fetch list of running systemd services etc)